engine/src/main/java/org/terasology/math/Region3i.java

```diff
@@ -51,6 +51,12 @@ public static Region3i createFromMinAndSize(BaseVector3i min, BaseVector3i size)
         return new Region3i(min, size);
     }
 
+    /**
+     * Create a region with center point and x,y,z coordinate extents size
+     * @param center the center point of region
+     * @param extents the extents size of each side of region
+     * @return a new region base on the center point and extents size
+     */
```

Comment:
all the createxxx methods are useful to be documentet.
Take care of createFromMinAndSize at line47 too. It would be nice to know that this could return an empty region if the size is negative.